package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
a security vulnerability issue with one of the packages:

>  === npm audit security report ===                        
> 
> # Run  npm install react-scripts@4.0.1  to resolve 1 vulnerability
> SEMVER WARNING: Recommended action is a potentially breaking change
> 
>   High            Prototype Pollution                                           
> 
>   Package         object-path                                                   
> 
>   Dependency of   react-scripts                                                 
> 
>   Path            react-scripts > resolve-url-loader > adjust-sourcemap-loader  
>                   > object-path                                                 
> 
>   More info       https://npmjs.com/advisories/1573                             
> 
> 
> 
> found 1 high severity vulnerability in 1618 scanned packages
>   1 vulnerability requires semver-major dependency updates.

Reply:
Done